mindmeld/components/entity_recognizer.py

```diff
@@ -176,25 +175,19 @@ def load(self, model_path):
         logger.info(
             "Loading entity recognizer: domain=%r, intent=%r", self.domain, self.intent
         )
-        try:
-            er_data = joblib.load(model_path)
-
-            self.entity_types = er_data["entity_types"]
-            self._model_config = er_data.get("model_config")
 
-            # The default is True since < MM 3.2.0 models are serializable by default
-            is_serializable = er_data.get("serializable", True)
+        # underlying model specific load
+        self._model = load_model(model_path)
 
-            if is_serializable:
-                # Load the model in directly from the dictionary since its serializable
-                self._model = er_data["model"]
-            else:
-                self._model = create_model(self._model_config)
-                self._model.load(model_path, er_data)
-        except (OSError, IOError) as e:
-            msg = "Unable to load {}. Pickle file cannot be read from {!r}"
-            raise ClassifierLoadError(msg.format(self.__class__.__name__, model_path)) from e
+        # classifier specific load
+        try:
+            er_data = pickle.load(open(self._get_classifier_resources_save_path(model_path), "rb"))
```

Comment:
Do we have to use pickle or can we still use joblib?

Reply:
I did some quick benchmarking and found that pickle is better time-wise, although using either makes no difference otherwise.